<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project_guide</class>
 <widget class="QDialog" name="Project_guide">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>658</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>640</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="title">
    <string>Project  Property </string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Root Directory Project Property</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="3" column="0">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Maximum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>30</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="checkBox_6">
                   <property name="text">
                    <string>Static Library Project</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <layout class="QGridLayout" name="gridLayout_4">
                 <item row="0" column="0">
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeType">
                    <enum>QSizePolicy::Maximum</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>30</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="checkBox_5">
                   <property name="text">
                    <string>Dynamic Library Project</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBox_3">
               <property name="text">
                <string>Library Project</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBox_7">
               <property name="text">
                <string>Executable Project</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Sub Direcotry Project Property</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Dynamic Library Project</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>Static Library Project</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_13">
    <property name="horizontalSpacing">
     <number>45</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
